src/backend/parser/parse_func.c

```diff
@@ -2062,6 +2062,10 @@ funcname_signature_string(const char *funcname, int nargs,
 
 	initStringInfo(&argbuf);
 
+	if (sql_dialect == SQL_DIALECT_TSQL && remove_db_name_in_schema_hook) {
+		funcname = (*remove_db_name_in_schema_hook)(funcname);
```

Comment:
This is dealing with funcname, is schema name mapping needed here? 
I'm guessing it's something like `db_schema.func()` -> `schema.func()`, but is there always a schema involved?

Reply:
No, thanks for pointing this out. I'll add a check to determine if there are any schema changes.

Reply:
Is this checked in new code?

Reply:
Would you please check if `funcname` can possibly don’t contain schema name? This is the only place that doesn’t explicitly state that there must be a schema name.
- If it’s guaranteed to have schema name, then your current implementation is okay, may be better to mention that it’s caller’s responsibility to make sure that input object_name contains a schema name
- If it’s possible to have one-part funcname without schema, then you may want to add special handling for this scenario